### ISSUE TITLE
Change default drop_last=False to True

### ISSUE THREAD

**Issue by felix-e-h-p** [openclimatefix/PVNet#528]
This will remove last incomplete batch and guarantees every batch has exactly specified batch_size.

This prevents BatchNorm from ever seeing batch size=1 and related errors when training.

https://github.com/openclimatefix/pvnet/blob/main/pvnet/datamodule.py#L69

**Comment by dfulu** [comment]
Solved by #529
